concepts/comparison/about.md

```diff
@@ -27,7 +27,7 @@ The result of the comparison is always a boolean value, so either `true` or `fal
 
 The comparison operators above can also be used to compare strings.
 In that case, a dictionary (lexicographical) order is applied.
-The ordering is "by unicode codepoint value".
+The ordering is _by unicode codepoint value_.
```

Comment:
I had quoted this because it's a copy-paste from the manual, but I didn't link it. How about
```suggestion
The ordering is [by unicode codepoint value][sort].

[sort]: https://stedolan.github.io/jq/manual/v1.6/#sort,sort_by(path_expression)
```